forms/forms_func.py

```diff
@@ -704,6 +705,11 @@ def hosp_get_operation_data(num_dir):
                 if field[3] == 'Оперативное вмешательство':
                     operation_data['plan_operation'] = field[2]
                     continue
+                if field[3] == 'Категория сложности':
+                    category_difficult = f"<br/><font face=\"PTAstraSerifBold\" size=8.7>(Сложность - {field[2]})</font>"
```

Comment:
Обычно везде мы писали атрибуты с кавычками
```suggestion
                    category_difficult = f"<br/><font face=\"PTAstraSerifBold\" size="8.7">(Сложность - {field[2]})</font>"
```

Reply:
Лучше всегда делать одинаково